### DISCUSSION TITLE
type of col in QTable's props.col

### DISCUSSION THREAD

**Discussion by dajermstad** [quasarframework/quasar#12276] [General - Components / Directives / etc]
[I'am]([url](url)) using TS in my project and get 'Object is of type unknown' and a red squigly line on the col object when creating an QTable and looping the columns. How can I set type on col object?

` <q-th v-for="col in props.cols" :key="col.name" :props="props" >
                {{ col.label }}
              </q-th>`

Best regards

**Comment by topvis** [comment]
Import the type definition from quasar:
```Javascript
import type { QTableColumn } from 'quasar'
```
Then use it in template:
```HTML
<q-th v-for="col in (props.cols as QTableColumn[])" :key="col.name" :props="props" > {{ col.label }} </q-th>
```
